### DISCUSSION TITLE
Classify_crown_pc

### DISCUSSION THREAD

**Discussion by Sylva3D** [lmterryn/ITSMe#3] [Q&A]
Hi, 
Thanks for sharing the ITSMe package, that is very usefull for the lidar & forest community. 

I notice that when I run the "classify_crown_pc" function on trees that don't have a crown (typically dead trunk with a missing top and no branches), the algorithm seems to loop without ever getting a result. 
This is quite annoying when running the algorithm in a loop on individual segmented trees that are automatically segmented because if there is a dead trunk among them, we will never get any results. 

Would it be possible to modify the script so that it runs anyway in this kind of situation? (ex: when it encounters a tree without any crown or branches, to get only the trunk classified). 

Thanks in advance

**Comment by lmterryn** [comment]
Thank you for your message! 
That is a good point, I will have a look at it and try to fix it asp.

**Reply by lmterryn** [reply]
@BastienUlg, I have made some adaptations to the classify_crown_pc code. Let me know if it works!

**Reply by Sylva3D** [reply]
@lmterryn Yes it's working now, thank you very much!
